asv.conf.json

```diff
@@ -0,0 +1,21 @@
+
+{
+    "version": 1,
+    "project": "stardis",
+    "project_url": "https://tardis-sn.github.io/tardis",
+    "repo": ".",
+    "install_command": ["in-dir={env_dir} python -mpip install {wheel_file}"],
+    "uninstall_command": ["return-code=any python -mpip uninstall -y {project}"],
+    "build_command": [
+        "python setup.py build",
+        "PIP_NO_BUILD_ISOLATION=false python -mpip wheel --no-deps --no-index -w {build_cache_dir} {build_dir}"
+    ],
+    "branches": ["main"],
+    "environment_type": "conda",
+    "show_commit_url": "https://github.com/tardis-sn/stardis/commit",
+    "conda_environment_file": "tardis_env3.yml",
```

Comment:
perhaps this as well? I don't know how stardis is installed so this might be correct

Reply:
Afaik stardis is installed after tardis and it's dependancies are installed